cypress/support/component.ts

```diff
@@ -1,3 +1,6 @@
+/* eslint-disable @typescript-eslint/no-namespace */
```

Comment:
probably not needed

Reply:
It is required for the global namespace definition on line $%

Reply:
ah, ok, thank you